cpp/cmake_modules/FindPythonLibsNew.cmake

```diff
@@ -234,11 +234,7 @@ FUNCTION(PYTHON_ADD_MODULE _NAME )
       SET_TARGET_PROPERTIES(${_NAME} PROPERTIES LINK_FLAGS
                           "-undefined dynamic_lookup")
     ELSE()
```

Comment:
On Unix system at least, you should only link to `libpython` if you embed the Python interpreter directly. Otherwise the needed symbols are provided by the runtime (i.e. the `python` executable). For being `manylinux1` compliant, one is not allowed to link to `libpython`. Partly because not all systems ship it, also because the interpreter ABI/API was not stable between patch releases for Python<3.5.